Mathlib/MeasureTheory/Group/LIntegral.lean

```diff
@@ -53,6 +53,21 @@ theorem lintegral_div_right_eq_self [IsMulRightInvariant μ] (f : G → ℝ≥0
 
 end MeasurableMul
 
+section MeasurableInv
+
+variable [Group G] [MeasurableInv G]
+
+/-- Involuting a function does not change its Lebesgue integral with respect to a
+inverse-invariant measure. -/
+@[to_additive]
```

Comment:
Please add a docstring to the additive version as well (with `@[to_additive "here is a docstring"]`)